libplum/libplum.c

```diff
@@ -0,0 +1,8490 @@
+#include <stdio.h>
```

Comment:
The files don't specify which license they have. Either we talk to the author to add a header specifying the license, or we copy this file https://github.com/aaaaaa123456789/libplum/blob/master/LICENSE to the repository to the libplum folder.

Reply:
They don't have a copyright header because they aren't copyrighted — copyright is disclaimed. If you want to edit your copy of the files to add a header stating this, you can, obviously.

Reply:
Then yes, we should add a header that follows this system:

https://github.com/AntonioND/gba-bootstrap/blob/63e6183501239d9ea1abbe007c5d18913d963157/template_c/source/sys/gba_crt0.s#L1-L3

It's quite hard to remember the license of every file in the project (I'm not just talking about grit) so it's important to have a per-file header that tells you what's the license of that specific file. It took me ages to clarify the licensing status of all files in libnds, for example, and it exposed a few files that weren't licensed like the rest.

Reply:
Added a LICENSE file to the directory.

Reply:
Looks good to me.